app/templates/home/premium.hbs

```diff
@@ -0,0 +1,22 @@
+<h1>DATAFRUITS PREMIUM</h1>
+<h2>Support a listener and DJ owned and ran internet radio station</h2>
+
+$7 / mo - $70 / yearly !
+
+<img src="/assets/images/df_premium.gif" alt="{{ t "premium.membership_card" }}" />
```

Comment:
I'm seeing a missing translation error for `premium.membership_card` on the `/premium` page